.drone.yml

```diff
@@ -107,6 +107,28 @@ pipeline:
       event: [push, tag]
       branch: [dev]
 
+  compile-gource-video:
+    repo: kowalatech/kcoin
+    secrets: [ docker_username, docker_password ]
+    dockerfile: gource/Dockerfile
```

Comment:
The pipeline will need an `image:` tag, which is why the CI is complaining. To make it work, we need a docker image on DockerHub. If there isn't one already, then the gource/Dockerfile should be split out to a different repo and pushed to dockerhub on commit.